eBay-brand-tracker/ebayBrandTracker/ebayBrandTracker/spiders/ebaybtspider.py

```
@@ -40,10 +46,10 @@ def parse(self, response):
         # Check if there is a next page and follow it
         next_page = response.css('a.pagination__next::attr(href)').get()
         if next_page:
-            # print('**********NEXT PAGE URL**********')
-            # print(next_page)
-            # next_page_url = f"https://www.ebay.com{next_page}"
-            yield scrapy.Request(url=next_page, callback=self.parse, meta={'brand': brand})
+            headers = {
```

Ruff

Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)